.github/workflows/github-actions-cron-update-yosys.yml

```diff
@@ -0,0 +1,43 @@
+name: Create draft PR for updated YOSYS submodule
+on:
+  schedule:
+  - cron: "0 8 * * MON"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository code recursively
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - name: Pull changes in yosys submodule
+        id: remote-update
+        run: |
+          cd tools/yosys
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git config --get remote.origin.fetch
+          echo "::set-output name=has_update::$(git --no-pager diff --name-only origin/master..HEAD)"
```

Comment:
@vvbandeira do you think I should check the difference between the head and the latest tag not the master? or it wouldn't matter?

Reply:
since we are going to update to the tag (going from Matt's comment earlier) it makes sense